packages/shared/src/utils/constants/idv-failure-codes.ts

```diff
@@ -1,7 +1,256 @@
-export const idv_error_statuses = Object.freeze({
-    poi_name_dob_mismatch: 'POI_NAME_DOB_MISMATCH',
-    poi_dob_mismatch: 'POI_DOB_MISMATCH',
-    poi_name_mismatch: 'POI_NAME_MISMATCH',
-    poi_expired: 'POI_EXPIRED',
-    poi_failed: 'POI_FAILED',
+export const IDV_ERROR_STATUS = Object.freeze({
+    DobMismatch: {
+        code: 'DobMismatch',
+        message: 'The date of birth retrieved from your document doesn’t match your profile.',
+    },
+    DocumentRejected: { code: 'DocumentRejected', message: 'Document was rejected by the provider' },
+    EmptyStatus: {
+        code: 'EmptyStatus',
+        message: 'The verification status was empty, rejected for lack of information.',
+    },
+    Expired: { code: 'Expired', message: 'The document’s validity has been expired.' },
+    InformationLack: {
+        code: 'InformationLack',
+        message: 'The verification is passed but the personal info is not available to compare.',
+    },
+    MalformedJson: {
+        code: 'MalformedJson',
+        message: 'The verification status is not available, provider says: Malformed JSON.',
+    },
+    NameMismatch: {
+        code: 'NameMismatch',
+        message: 'The name retrieved from your document doesn’t match your profile.',
+    },
+    RejectedByProvider: {
+        code: 'RejectedByProvider',
+        message: 'The document was rejected by the Provider.',
+    },
+    Underage: {
+        code: 'Underage',
+        message: 'You’re under legal age.',
+    },
+    Deceased: {
+        code: 'Deceased',
+        message: 'The document’s owner is deceased.',
+    },
+    Failed: {
+        code: 'Failed',
+        message: 'We were unable to verify the identity document with the details provided.',
+    },
+    NameDOBMismatch: {
+        code: 'NameDOBMismatch',
+        message: '',
```

Comment:
don't we have any message for `NameDOBMismatch`?

Reply:
No. It will be a custom message